### PR TITLE
Upgrade python

### LINKED ISSUE #52: Migrate to python 3.11
We have planned to migrate exodus to debian 12 and python 3.11.
It's can be good also to update exodus-standalone.
Files need to be updated:
- [ ] Workflow Github
- [ ] DockerFile

### PR DESCRIPTION
**Pull request by Jean-BaptisteC**
Fixes #52

This PR:
- Update python to 3.11
- Update exodus-core
- Update dexdump

### CODE REVIEW

**Review by codeurimpulsif — APPROVED** [review]
Seems ok for me, but wait for @pnu-s review.

File: requirements.txt [hunk 1]
```diff
@@ -1 +1 @@
-exodus-core==1.3.9
+exodus-core==1.3.11
```
Comment (pnu-s) [inline requirements.txt:1]:
```suggestion
exodus-core==1.3.12
```

**Comment by Jean-BaptisteC** [comment]
Tested in local (not with Docker)
image

**Comment by erfan125** [comment]
> Fixes #52
> 
> This PR:
> - Update python to 3.11
> - Update exodus-core
> - Update dexdump

**Pull request merged**